charts/histogram/src/ext.js

```diff
@@ -19,8 +19,8 @@ export default function ext(env) {
       sharing: true,
       viewData: true,
     },
-    importProperties: null, // Prevent conversion to and from this object
-    exportProperties: null,
+    importProperties: conversion.hypercube.importProperties,
+    exportProperties: conversion.hypercube.exportProperties,
```

Comment:
this will not work you need to change how the arguments are passes on to 'qlik-object-conversion'